docs/architecture/decisions/adr-004.md

```diff
@@ -0,0 +1,33 @@
+[accessibility support baseline]: https://www.w3.org/TR/WCAG-EM/#step1c
+
+# ADR 4: Model browser specific values and computations
+
+## Context
+
+Having decided on [ADR 2](adr-002.md), we will no longer be tied to any one specific browser and are left with the choice of what to do about things that browsers are not aligned on. For example, if Alfa needs to implement an API to perform a given task and a given set of browsers do not agree on how the API should be implemented, Alfa can do several things:
+
+1. Implement the API the way it is implemented in the _least_ capable browser, in terms of the API, among a given set of browsers.
+
+2. Implement the API the way it is implemented in the _most_ capable browser, in terms of the API, among a given set of browsers.
+
+3. Provide several implementations of the API that cover all the different implementations of the API among a given set of browsers.
+
+Our proprietary accessibility conformance testing engine at Siteimprove takes the second approach in practice as it is usually run within the latest version of the Chrome browser and relies heavily on browser APIs. Other tools that operate with the concept of an [accessibility support baseline][] will typically take a combination of the first and second approach where some browser APIs are used and then others implemented manually according to the capabilities of the [accessibility support baseline][].
+
+The quality shared by the first and second approach is that they both produce just a single implementation of the API, though with potentially different capabilities. However, both approaches suffer from the fact that depending on how the API is implemented, an accessibility issue could be found in either the least or most capable browser, while being missed in the other. As an example, a contrast issue might appear in an older browser that does not support a specific color format, while the contrast would be perfectly fine in a newer browser that does support the specific color format. The third approach avoids this issue entirely by implementing the API in all possible ways for a given set of browsers.
```

Comment:
> a contrast issue might appear in an older browser that does not support a specific color format, while the contrast would be perfectly fine in a newer browser

I am assuming that older and newer here refer to "least capable" and "most capable"?